mage/Editors/FormPalette.cs

```diff
@@ -294,21 +301,43 @@ private unsafe void DrawColorBar()
         private unsafe void DrawLine(BitmapData imgData, int color)
```

Comment:
Is it possible to call this function if the palette editor is open when the theme changes?